<launch>
  
  <node pkg="sr_slam" type="pos_msg" name="pos_msg" output="log">
    <param name="record_name"                 value="./msg_pos/pos_2D_record_21.log"/>
    <param name="record_name"                 value="./msg_pos/pos_2D_record_eit.log"/>
    <param name="record_name"                 value="./msg_pos/pos_2D_record_eit_f5.log"/>

  </node>

  <node pkg="vis_navigation" type="vis_navigation" name="slam_ui" output="log">
  <param name="start_u"                       value="2680" />  <!-- 760 -->
  <param name="start_v"                       value="550" /> <!-- 1150 -->
  <param name="u_resolution"                  value="0.01960243"/>
  <param name="v_resolution"                  value="0.01960243"/>
  <param name="demo_place"                    value="eit_f5"/> <!-- default: etas_f5 -->

  <!-- large transformation thresholds -->
  <param name="large_x_threshold"             value="60"/>
  <param name="large_y_threshold"             value="140"/>
  <!-- param name="large_theta_threshold"         value="2"/ -->

  <param name="record_pos_2D"                 value="false"/> <!-- whether to record pos msg from rgbdslam -->

  <!-- mapping_rule = 1, -M_PI/2, mapping_rule = 2, 0, mapping_rule = 3, M_PI/2 -->
  <!-- param name="init_heading"              value="0" --> <!-- default: -M_PI/2 -->
  <param name="init_heading"                  value="-1.570796327"/> <!--  3.141592654 default: -M_PI/2 -->

  <param name="bg_img_name"                   value="/home/davidz/work/ros_hydro_ws/devel/lib/vis_navigation/EIT_Building_Floor5_2.png"/>
  <!-- line extractor parameters -->
  <param name="number_of_points"              value="50"/>  <!-- the minimal number of points in a line -->
  <param name="min_hist_delete"               value="25"/>   <!-- 5 threshold for deleting points-->
  <param name="min_hist_thre"                 value="45"/>  <!-- 15 threshold for hist filter --> 
  <param name="min_line_length"               value="10000"/> <!-- 10000 = 100^2, 100*0.02 = 2m sq of the minimal line length -->

  <!-- save hist and line points -->
  <param name="save_hist_points"              value="false"/>
</node>


</launch>
